src/CommunityToolkit.Maui/Behaviors/ICommunityToolkitBehavior.shared.cs

```diff
@@ -89,7 +89,7 @@ protected void OnViewPropertyChanged(object? sender, PropertyChangedEventArgs e)
 		}
 		catch (Exception ex) when (Options.ShouldSuppressExceptionsInBehaviors)
 		{
-			Trace.WriteLine(ex);
+			Trace.TraceError(ex.StackTrace);
```

Comment:
I think we should rely on the full information

```suggestion
			Trace.TraceError(ex);
```